.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+indent_style = space
+indent_size = 4
```

Comment:
I don't know others' takes on this, but as for me *personally*, I prefer 2-space indentation. What would you think about everything to tabbed, and then individuals can specify in their IDEs the size of the tabs? Consistent in the codebase, and still accommodates personal preferences.